schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id integer,
```

Comment:
- [ ] [OPTIONAL] I would suggest that you make this id column  PRIMARY KEY and also give it NOT NULL IDENTITY

Reply:
@BertrandConxy Thank you